lib/ansible/module_utils/compat/selinux.py

```diff
@@ -62,7 +62,7 @@ def from_param(cls, strvalue):
         fn.restype = cfg.get('restype', c_int)
 
         # just patch simple directly callable functions directly onto the module
-        if not fn.argtypes or not any(argtype for argtype in fn.argtypes if type(argtype) == base_ptr_type):
+        if not fn.argtypes or not any(argtype for argtype in fn.argtypes if isinstance(argtype, base_ptr_type)):
```

Comment:
@nitzmahone Was there a reason for this to use `==` originally instead of `is` or `isinstance`?

Reply:
Yeah, just changing the `==` to `is` is probably the right thing in this case- `isinstance` is expensive and unnecessary here.

Reply:
(I don't recall why I had to do all this silliness in the first place, but replacing the `==` with `is` should be semantically equivalent, where `isinstance` could potentially change the behavior in significant ways.

Reply:
Should we ignore `C0123` altogether then?

Reply:
No need to ignore it, just replace `==` with `is`, since it's equivalent and passes the test.